### DISCUSSION TITLE
How should i delete a reporitory which is unavailable due to DMCA takedown?

### DISCUSSION THREAD

**Discussion by yaohwu** [community/community#21933] [New to GitHub]
<p>How should i delete a reporitory which is unavailable due to DMCA takedown ? </p>
<p>A repository like this <a href="https://github.com/yaohwu/StarCraft" rel="nofollow noopener">https://github.com/yaohwu/StarCraft</a>.</p>

**Comment by mpboom** [comment]
<p>I think you might want to contact GitHub support for that.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>As <a class="mention" href="/mpboom">@mpboom</a> stated, for situations where you are having problems related to your specific GitHub account, you will want to contact GitHub support directly by emailing <a href="mailto:support@github.com">support@github.com</a> or <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>.</p>
<p>The community forum is designed to be helpful in soliciting community advice for more general topics and collaborating on projects.</p>

**Reply by muescha** [reply]
with github.com/contact there is no way to send a message to support :(

**Reply by ThirdEyeRose** [reply]
Is there a "my tickets" in the top-right hand corner?

![image](https://user-images.githubusercontent.com/1864667/208933971-ea59197c-871e-4393-a7d5-3948aff1096b.png)

When I clicked on that, I was able to submit a new ticket. However, since I no longer work at GitHub, I have no idea if that is available for everyone

**Reply by muescha** [reply]
thx - i find you that i tried the category [submitting-content-removal-requests](https://docs.github.com/github/site-policy/submitting-content-removal-requests)  (which i think is the right category

i had to go to some trough some other categories and finally find there a contact form. this was the tricky part to go to an unrelated category and open there a ticket.

**Comment by yaohwu** [comment]
<p>thanks :robotembarrassed:</p>
